bq-connector/docai_bq_connector/connector/BqDocumentMapper.py

```diff
@@ -247,6 +247,7 @@ def _map_document_to_bigquery_schema(
     ):
         result: dict = {}
         for field in fields:
+            print(f"field={field}")
```

Comment:
I think this can be removed, or change to use logging to be consistent with the other py files.

Reply:
Indeed, debug print - fixed, good catch